Assertion fails and aborts compile commands generation

Thanks for this great tool!

For specific targets I get an assertion error here: https://github.com/hedronvision/bazel-compile-commands-extractor/blob/1d21dc390e20ecb24d73e9dbb439e971e0d30337/refresh.template.py#L90

```

AssertionError: Something went wrong in makefile parsing to get headers. First entry should be the source file. Output: 
interpolator_fuzz_test.o: \
  external/host_ros_clang_toolchain/lib/clang/11.0.1/share/asan_blacklist.txt \
  test/interpolator_fuzz_test.cpp \
...
```

Not sure, but it seems that the source file is not the first but the second entry?

--- a/refresh.template.py
+++ b/refresh.template.py
@@ -68,7 +68,7 @@ def _get_headers(compile_args: typing.List[str], source_path_for_sanity_check: t
     # Strip sanitizer ignore lists...so they don't show up in the dependency list.
     # See https://clang.llvm.org/docs/SanitizerSpecialCaseList.html and https://github.com/hedronvision/bazel-compile-commands-extractor/issues/34 for more context.
     header_cmd = (arg for arg in header_cmd
-        if not arg.startswith('-fsanitize-ignorelist='))
+        if not arg.startswith('-fsanitize'))
 
     # Dump system and user headers to stdout...in makefile format, tolerating missing (generated) files
     # Relies on our having made the workspace directory simulate the execroot with //external symlink

